src/main/scripts/startMZmine_Windows.bat

```diff
@@ -1,12 +1,44 @@
 @echo off
 
-rem This is necessary to access the TOTAL_MEMORY and ADDRESS_WIDTH variables inside the IF block
+:: *****************************************
+:: Optional values - Please modify as needed
+:: *****************************************
+
+:: Total amount of memory in MB available to MZmine 2.
+:: AUTO = automatically determined
+:: Default: AUTO
+set HEAP_SIZE=AUTO
+
+:: Location where temporary files will be stored.
+:: Default: %TEMP%
+set TMP_FILE_DIRECTORY=%TEMP%
+
+:: Allow MZmine to send anonymous statistics on the module usage? 
+:: 1 = activated, 0 = deactivated
+:: Default: 1
+set MZMINE_STATISTICS=1
+
+:: It is usually not necessary to modify the JAVA_COMMAND parameter, but if you like to run
+:: a specific Java Virtual Machine, you may set the path to the java command of that JVM
+set JAVA_COMMAND=java

```

Comment:
Since you're cleaning up the scripts, maybe we could change this to 
JAVA_COMMAND=java.exe
in the Windows script. That will make the meaning more obvious. The comment can be also updated ("..., you may set the path to the java.exe command...").